SU-SWS/linky_clicky
# SITES-346: Block Backup and Migrate restorations

# READY FOR REVIEW

# Summary
- Ensure that User 1 and administrators cannot restore a website via the Backup and Migrate module.

# Needed By (Date)
- 2.15 end of sprint would be nice; not killer if it rolls over to next sprint.

# Criticality
- How critical is this PR on a 1-10 scale? 6
- Affects ACSF

# Steps to Test

0. Create a site locally (any product should work)
1. Download the `paranoia` module
2. Apply the [patch in comment #29](https://www.drupal.org/files/issues/paranoia-block_additional_forms-2313945-29.patch)
3. Run `php-administrator.feature` and `php-userone.feature` and ensure that all Scenarios pass

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules? ACSF

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/SITES-346
- Other PRs: https://github.com/SU-SWS/acsf-cardinald7/pull/22
- Anyone who should be notified? ( @sherakama )
